Azure/sonic-utilities
# Allow fw update for other boot type against on the previous "none" boot fw update

https://github.com/Azure/sonic-buildimage/issues/8928
https://github.com/Azure/sonic-buildimage/issues/8926
https://github.com/Azure/sonic-buildimage/issues/8925
https://github.com/Azure/sonic-buildimage/issues/8924

#### What I did
Allow fwutil update all for other boot type if any previous fw update done for "none" boot type

#### How I did it
Allow fwutil update all for other boot type if any previous fw update done for "none" boot type

#### How to verify it
1. Run fwutil update all for boot_type="none"
2. Run fwutil update all for any other boot_type
3. Verify if the 2nd update is proceeded.
#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
